lib/pdf_fill/filler.rb

```diff
@@ -102,7 +102,7 @@ def combine_extras(old_file_path, extras_generator, form_class)
 
         merge_pdfs(old_file_path, extras_path, file_path)
         # Adds links and destinations to the combined PDF
-        if extras_generator.try(:section_coordinates)
+        if extras_generator.try(:section_coordinates) && !extras_generator.section_coordinates.empty?
```

Comment:
@rilorie I think this change makes sense as any form that uses the V2 extras generator will always have a `section_coordinates` array, but it might not be populated for every form. I was getting an error [here](https://github.com/department-of-veterans-affairs/vets-api/blob/8631dacc/lib/pdf_fill/pdf_post_processor.rb#L36) without this change. Thoughts?